babel.config.js

```diff
@@ -121,14 +121,28 @@ module.exports = {
             resolvePath,
           },
         ],
-        'transform-react-constant-elements',
-        'transform-dev-warning',
-        ['react-remove-properties', { properties: ['data-mui-test'] }],
-        ['transform-react-remove-prop-types', { mode: 'remove' }],
+        [
+          '@babel/plugin-transform-runtime',
+          {
+            corejs: 2,
```

Comment:
Reproduce https://github.com/zeit/next.js/blob/master/packages/next/build/babel/preset.ts#L75

Reply:
Alternatively, we could try to use next/babel preset: 
https://github.com/zeit/next.js/blob/canary/examples/with-custom-babel-config/.babelrc